background/background.js

```diff
@@ -0,0 +1,11 @@
+const backgroundPath = {
```

Comment:
I think calling this file 'backgroundsMapping' could be better (you would know what's inside at the first glance).

Reply:
Done